packages/components/src/support/types/index.ts

```diff
@@ -14,6 +14,8 @@ export type InputState = 'success' | 'warning' | 'error';
 
 export type InputVariant = 'small' | 'medium' | 'large';
 
+export type SelectState = InputState;
```

Comment:
I don't see a benefit in defining SelectState type, InputState sounds fine for me even for select input, but nothing wrong of course.

Reply:
Removed. (Of course, another point of view is: select is also input in general)